test/Wrappers/wrapper-erc20.test.ts

```diff
@@ -0,0 +1,72 @@
+import {ethers} from "hardhat"
+import { expect } from "chai"
+import { ERC20Wrapper } from "../../lib/wrappers/tokens/erc20"
+import { ERC20Mock } from "../../typechain-types"
+
+describe("ERC20 Wrapper", async () => {
+    let tokenContract: ERC20Mock
+
+    beforeEach(async () => {
+        const [deployer] = await ethers.getSigners()
+
+        tokenContract = await ethers.deployContract("ERC20Mock", [deployer, 100, "Mock", "MKT"])
+    })
+
+    it("should show token info", async () => {
+        const tokenWrapper = new ERC20Wrapper(
+            tokenContract.target, ethers.provider
+        )
+
+        expect(await tokenWrapper.symbol())
+            .to.be.equal("MKT")
+        expect(await tokenWrapper.name())
+            .to.be.equal("Mock")
+        expect(await tokenWrapper.decimals())
+            .to.be.equal("18")
+        expect(await tokenWrapper.totalSupply())
+            .to.be.equal("100")
+    })
+    it("should transfer", async () => {
+        const [deployer, target] = await ethers.getSigners()
+
+        const tokenWrapper = new ERC20Wrapper(
+            tokenContract.target, ethers.provider
+        )
```

Comment:
If wrapper it is needed try to create it in a `beforeEach()` hook and ideally with a fixture, [here an example](https://github.com/Fusyona/web3-core/blob/7de6dc7ca04f0db322b13211913db41f21761940/test/Cooldown.ts?plain=1#L8-L21)